packages/react/.storybook/main.ts

```diff
@@ -30,16 +30,12 @@ const config: StorybookConfig = {
       ICONS: icons.join(","),
     };
   },
-  addons: [
-    getAbsolutePath("@storybook/addon-essentials"),
-    getAbsolutePath("storybook-addon-jsx"),
-    getAbsolutePath("@storybook/addon-a11y"),
-  ],
+  addons: ["@storybook/addon-essentials", "storybook-addon-jsx", "@storybook/addon-a11y"],
   stories: ["../src/**/*.stories.@(ts|tsx)"],
   previewHead: (head) => `
     ${head}
-    <link rel="stylesheet" href="dso-toolkit/dist/dso.css">
-    <script src="iframe-resizer/child/index.umd.js"></script>
+    <link rel="preload" href="/static/dso-toolkit/assets/fonts/Asap/Asap-Italic-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
+    <link rel="preload" href="/static/dso-toolkit/assets/fonts/Asap/Asap-VariableFont_wdth,wght.ttf" as="font" type="font/ttf" crossorigin>
```

Comment:
In de React Storybook:
- `dso.css` is verplaatst naar `preview.ts`
- Net zoals iframe-resizer child
- Root path voor bundled assets is `/static/dso-toolkit/assets`, geen idee waarom. Zal wel de Storybook Webpack configuratie zijn.